### DISCUSSION TITLE
Overwrite helm deployed application via argocd

### DISCUSSION THREAD

**Discussion by ajithvatsanp** [argoproj/argo-cd#16395] [Q&A]
Hi,

In our K8S cluster, we already deployed an application via helm. Without deleting it, we have redeployed the same application via argocd. Now we can see that, the pod has been recreated at the time of argocd deployment. But related configmaps, services and even replica set seems to be stick with old helm way. 

The service label still points like below (still managed by helm):
Labels:         app.kubernetes.io/instance=artifactory-proxy
                   **app.kubernetes.io/managed-by=Helm**
                   app.kubernetes.io/name=artifactory-proxy
                   app.kubernetes.io/version=1.20.2-alpine
                   argocd.argoproj.io/instance=artifactory-proxy
                   helm.sh/chart=artifactory-proxy-0.4.3

Is this expected behavior? Still I could see helm release present there and helm values can also be retrieved from there.
Please enlighten us more regarding the above behavior. More views are welcome.

**Comment by FernandoMiguel** [comment]
i would not expect argo deployments to interact in anyway with existing deployments.
you can adopt existing ones, by adding the required annotations

**Reply by ajithvatsanp** [reply]
Indeed, we too expected the same. But when we do the deployment via argocd, we could see the mentioned behavior in the existing deployment.
